docs/source/release_notes/52_0.rst

```diff
@@ -0,0 +1,139 @@
+- Improved `avocado.utils.iso9660` with a more complete standard API
+  across all back-end implementations.
+- Improved `avocado.utils.build.make()`, which will now return the make
+  process exit status code.
+- Better error handling of the virtual machine plugin (`--vm-domain` and
+  related options).
+- The `avocado.utils.process` library makes it possible to ignore
+  background processes that never finish.
+- When discovering tests on a directory, the result now is a properly
+  alphabetically ordered list of tests.
+- Plugins can now be setup to run at a specific order.
+- A contrib script that looks for a job result directory based on a
+  partial (or complete) job ID is now available at
+  `contrib/scripts/avocado-get-job-results-dir.py`.
+- Support for filtering tests by user supplied “tags”.
+- Users can now choose to keep the complete set of files, including
+  temporary ones, created during an Avocado job run by using the
+  `--keep-tmp` option.
+- Tests running with the external runner (`--external-runner`) feature
+  will now have access to the extended behavior for SIMPLE tests, such
+  as being able to exit a test with the WARNING status.
+- Users will now be able to properly run tests based on any Unicode
+  string (as a test reference).
+- Test writers now have access to a test temporary directory that will
+  last not only for the duration of the test, but for the duration of
+  the whole job execution.
+- When running jobs with the TAP plugin enabled (the default), users
+  will now also get a results.tap file created by default in their job
+  results directory.
+- Avocado is now available on Fedora.
+- Avocado packages on PyPI now are always the last version.
+- Avocado test writers can now use a family of decorators, namely
+  `avocado.skip()`, `avocado.skipIf()` and `avocado.skipUnless()` to
+  skip the execution of tests.
+- Sysinfo collection based on command execution now allows a timeout to
+  be set.
+- The non-local runner plugins are now distributed in separate packages.
+- The `avocado.Test` class now better exports (and protects) the core
+  class attributes members (such as params and runner_queue).
+- Users sending SIGTERM to Avocado can now expect it to be properly handled.
+- `avocado.utils.linux_modules` functions now returns module name, size,
+  submodules if present, filename, version, number of modules using it,
+  list of modules it is dependent on and finally a list of params.
+- `avocado.TestFail`, `avocado.TestError` and `avocado.TestCancel` are
+  now public Avocado Test APIs, available from the main avocado namespace.
+- Support for new type of test status, `CANCEL`, and of course the
+  mechanisms to set a test with this status.
+- Introduction of the robot plugin, which allows Robot Framework tests
+  to be listed and executed natively within Avocado.
+- A brand new ResultsDB plugin.
+- When users run `avocado list --loaders ?` they used to receive a
```

Comment:
also this is a minor thing so I'd just mention that the listing of supported loaders (`--loader \?`) format was changed (or improved)